en/ros/ros2_comm.md

```diff
@@ -177,8 +177,8 @@ We can do this by running the bridge against PX4 running in the simulator.
    accelerometer_integral_dt: 4739
    ```
 
-You can also verify the rate of the message using `rostopic hz`. <!-- how? -->
-For the case of `sensor_combined`:
+You can also verify the rate of the message using `ros2 topic hz`. <!-- how? -->
```

Comment:
Actually can you remove the `<!-- how? -->` now?

Reply:
Done